libmscore/figuredbass.h

```diff
@@ -152,12 +152,12 @@ class FiguredBassItem final : public Element {
       FiguredBassItem::Modifier MusicXML2Modifier(const QString prefix) const;
 
       // standard re-implemented virtual functions
-      virtual FiguredBassItem*      clone() const override     { return new FiguredBassItem(*this); }
-      virtual ElementType         type() const override      { return ElementType::INVALID; }
-      virtual void      draw(QPainter* painter) const override;
-      virtual void      layout() override;
-      virtual void      read(XmlReader&) override;
-      virtual void      write(XmlWriter& xml) const override;
+      FiguredBassItem*  clone() const override   { return new FiguredBassItem(*this); }
+      ElementType       type() const override         { return ElementType::INVALID; }
+      void              draw(QPainter* painter) const override;
+      void              layout() override;
+      void              read(XmlReader&) override;
+      void              write(XmlWriter& xml) const override;
```

Comment:
Actually, I think this type of alignment is overreaction ;-) Having more than one spaces between return type and function name is unnatural.

Reply:
I also think so, I don’t like any alignment at all, and especially this. Because it breaks sooner or later and everything becomes poorly readable or not consistent.  
But I try to keep everything consistent within the nearest code, everything is so tuned in this file, so I added these indents.

Reply:
Feel free to break the format for the entire file if you have time ;-)